src/compas_rhino/utilities/drawing.py

```diff
@@ -833,6 +869,12 @@ def draw_mesh(vertices, faces, name=None, color=None, disjoint=False, **kwargs):
     guid = add_mesh(mesh)
 
     if guid != System.Guid.Empty:
+        if vertexcolors:
+            try:
+                compas_rhino.set_mesh_vertex_colors(guid, vertexcolors)
```

Comment:
Not sure here, but what happens if user specifies `vertexcolors` but not `color`? is that even a valid combination?
From https://developer.rhino3d.com/api/RhinoScriptSyntax/#mesh-MeshVertexColors
```
Note, for each vertex, there must
  be a corresponding vertex color. If the value is None, then any
  existing vertex colors will be removed from the mesh
```